plugin/tools.py

```diff
@@ -452,3 +482,16 @@ def get_clang_version_str(clang_binary):
         else:
             raise RuntimeError(
                 " Couldn't find clang version in clang version output.")
+
+    @staticmethod
+    def get_unique_str(init_string):
+        """ Generate md5 unique sting hash given init_string """
+        import hashlib
+        return hashlib.md5(init_string.encode('utf-8')).hexdigest()
+
+    @staticmethod
+    def get_position_hash(view, position_in_file):
+        """ Generate md5 unique sting hash from position in the file """
+        unique_id = Tools.get_unique_str(
+            view.file_name() + str(position_in_file))
```

Comment:
Yeah, I thought about like an hour before :) Totally not necessary. I just got starting about a unique identifier, so that's the reason for the hash. I guess I will remove it, as it is more readable in debug output.